javascript/javascript3/week2/main/functions.js

```diff
@@ -0,0 +1,238 @@
+const takingData = async () => {
+  const response = await fetch(
+    "https://raw.githubusercontent.com/LCurcuma/LCurcuma.github.io/refs/heads/main/js3/cards.json"
+  );
+  const cards = await response.json();
+  return cards;
+};
+
+//function for setting desk
+
+function settingDesk() {
+  const currentLevel = document.getElementById("levels").value;
+
+  //"deleting" starting menu
+  startingMenu.setAttribute("style", "display:none;");
+
+  //checking the chosen level (it's form 0 to 5)
+  //level #0 - only 4 cards on desk
+  //level #1 - 8 cards
+  //level #2 - 12 cards
+  //level #3 - 16 cards
+  //level #4 - 20 cards
+  //level #5 - 26 cards
+  switch (currentLevel) {
+    case "0":
+      setDesk(2);
+      break;
+    case "1":
+      setDesk(4);
+      break;
+    case "2":
+      setDesk(6);
+      break;
+    case "3":
+      setDesk(8);
+      break;
+    case "4":
+      setDesk(10);
+      break;
+    case "5":
+      setDesk(13);
+      break;
+  }
+}
+
+//function for setting desk with some amount of cards
+function setDesk(amountOfCards) {
+  amountOfCardsOnTheDesk = amountOfCards * 2;
+
+  //showing container with count
+  countContainer.removeAttribute("style");
+
+  takingData().then((cards) => {
```

Comment:
The instructions for this week require you to use `async` and `await`. How can you change this line to accommodate it?